docker-compose.yaml

```diff
@@ -76,6 +76,7 @@ services:
       - applogs:/usr/app/demo/logs
     ports:
       - '5173:5173'
+      - '24678:24678'
```

Comment:
@bfmatei I'm no Vite expert, but without exposing this, there'd be tons of errors in the log

Reply:
Yup. Fixed it as well.